flo_ai/flo_ai/llm/aws_bedrock_llm.py

```
@@ -0,0 +1,265 @@
+            else:
+                result.append(msg)
+
+        return result
+
+    @trace_llm_call(provider='bedrock')
+    async def generate(
+        self,
+        messages: list[dict],
+        functions: Optional[List[Dict[str, Any]]] = None,
+        output_schema: Optional[Dict[str, Any]] = None,
+        **kwargs,
```

Ruff (0.15.2)

[warning] Unused method argument: `kwargs`

(ARG002)